Modern uaa integration tests

selzoc opened the pull request:
### What is this change about?

Use a modern UAA for integration tests

### Please provide contextual information.

See commit

### What tests have you run against this PR?

integration!

jpalermo commented:
@selzoc , I believe `ci/dockerfiles/integration/install-uaa.rb` is not used correct?

selzoc commented:
> @selzoc , I believe `ci/dockerfiles/integration/install-uaa.rb` is not used correct?

Ah yes I believe we moved that functionality into `src/bosh-dev/lib/bosh/dev/sandbox/services/uaa_service.rb`.

Removed in ecce4ec08845119df26bb87ba91b3f49658c69b2

cunnie reviewed (approved):
I like how you freshen UUA download URL every time.
